.github/workflows/lint.yml

```diff
@@ -1,14 +1,33 @@
-on: push
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: master
 name: Test and lint
 jobs:
   lint:
     runs-on: ubuntu-18.04
     steps:
-    - uses: actions/checkout@v1
-    - uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
-    - run: echo "${{ secrets.CONFIG_JSON }}" | base64 --decode > config.json
-    - run: npm ci
-    - run: npm run test
-    - run: npm run lint
+      - name: Checkout repo contents
+        uses: actions/checkout@v1
+
+      - name: Setup node.js
+        uses: actions/setup-node@v1
+          with:
+            node-version: 12.x
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Test (including W3C API tests)
+        if: ${{ secrets.CONFIG_JSON }}
+        run: |
+          echo "${{ secrets.CONFIG_JSON }}" | base64 --decode > config.json
```

Comment:
Instead of putting JSON-encoded strings in secrets, I liked the way @foolip [did it in validate repo](https://github.com/w3c/validate-repos/blob/master/.github/workflows/validate.yml#L23)

Reply:
Right but this is not the PR you're looking for :)

That is not what that I am trying to fix here. Plus that part works...

Reply:
fair enough :)